src/Util/Asserts.h

```diff
@@ -0,0 +1,30 @@
+//
+//  Asserts.h
+//
+//  Copyright (c) 2016 Open Whisper Systems. All rights reserved.
+//
+
```

Comment:
Sorry, this is what I meant about the .pch above.  I'm a fan of small, lean .pchs but this kind of asserts header is imo a strong candidate for inclusion, since ideally I'd like to use asserts in nearly every source file.

Anyhow, it sounds like that's out of the scope of this PR.  For now, I'll just fix by manually importing the header.